changelog/add-deposit-schedule-restriction-support

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Restricted deposit schedule fixes on settings page
```

Comment:
I think that this could be made clearer to merchants and developers.
```suggestion
Deposit schedule changes are disabled when an account has restrictions on it.
```